GVRf/Framework/jni/objects/scene_object_jni.cpp

```diff
@@ -213,6 +213,28 @@ Java_org_gearvrf_NativeSceneObject_isColliding(
     SceneObject* other_object = reinterpret_cast<SceneObject*>(jother_object);
     return scene_object->isColliding(other_object);
 }
+
+JNIEXPORT void JNICALL
+Java_org_gearvrf_NativeSceneObject_setLODRange(
+        JNIEnv * env, jobject obj, jlong jscene_object, jfloat min_range, jfloat max_range) {
+    SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
+    scene_object->setLODRange(min_range, max_range);
 }
 
+JNIEXPORT jfloat JNICALL
+Java_org_gearvrf_NativeSceneObject_getLODMinRange(
+        JNIEnv * env, jobject obj, jlong jscene_object) {
+    SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
+    return scene_object->getLODMinRange();
 }
+
+JNIEXPORT jfloat JNICALL
+Java_org_gearvrf_NativeSceneObject_getLODMaxRange(
+        JNIEnv * env, jobject obj, jlong jscene_object) {
+    SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
+    return scene_object->getLODMaxRange();
+}
+
```

Comment:
Normally, I would say, in a JNI file like this, you should declare a JNI function and then implement it. For setLODRange, getLODMinRange, and getLODMaxRange, you just provided the implementation here without declaration. However since it compiles and runs, then it should be fine even though it seems to me that in some environments it won't compile or at least it will show some warnings... Not quite sure about this.